chamber/storages/boto3.py

```diff
@@ -30,36 +33,39 @@ def force_bytes_content(content, blocksize=1024):
     return content, False
 
 
-def s3_presigned_download_url(url=None, file=None, expiration=3600):
-    if not url and not file:
-        raise ValueError('You must provide either a "url" or "file".')
-
-    if url:
-        return url
-    if not settings.AWS_S3_ON:
-        return file.url
-
-    s3 = boto3.client('s3', config=Config(region_name=settings.AWS_REGION, signature_version='s3v4'))
-    url = s3.generate_presigned_url(
-        'get_object',
-        Params={
-            'Bucket': file.storage.bucket_name,
-            'Key': file.storage.get_full_relative_path(file.name),
-        },
-        ExpiresIn=expiration,
-    )
-    return url
+def get_storage_class(s3_storage_class, default_storage_class=DefaultStorage):
+    return storage_class if settings.AWS_S3_ON else default_storage_class
```

Comment:
@matllubos Ještě tohle. Až bude zapnuté `AWS_S3_ON`, tak to žuchne, protože proměnná `storage_class` neexistuje :)